src/pages/guides/getting-started/installation.mdx

```diff
@@ -15,34 +15,58 @@ Please follow these links to the official installation steps for each.
 
 ## Installing the Nitric CLI
 
-### macOS
+<CodeGroup>
 
-`nitric` is available via [homebrew](https://brew.sh/)
-
-```bash
+```bash {{ title: 'macOS', label: 'nitric is available via <a href="https://brew.sh/">homebrew</a>' }}
 brew install nitrictech/tap/nitric
 ```
 
-### Windows
-
-`nitric` is available via [scoop](https://scoop-docs.vercel.app/)
-
-```
+```bash {{ title: 'Windows', label: 'nitric is available via <a href="https://scoop.sh/">scoop</a>' }}
 scoop bucket add nitric https://github.com/nitrictech/scoop-bucket.git
 scoop install nitric
 ```
 
-### Linux
+```bash {{ title: 'Linux', label: 'nitric can be downloaded as a scripted install via curl.' }}
+curl https://nitric.io/install | bash
+```
+
+```bash {{ title: 'Brew', label: 'nitric is available via <a href="https://brew.sh/">homebrew</a>' }}
```

Comment:
Does brew need its own tab given it's provided under MacOS?

Reply:
This is a good point, worth thinking about as brew can also be used on Linux as well if users choose.

Reply:
^ yeah I added it for linux users. But then I guess they have the install script too. I can move it as a part of the linux one?